internal/k8s/status.go

```diff
@@ -560,6 +652,21 @@ func (su *statusUpdater) generateExternalEndpointsFromStatus(status []api_v1.Loa
 	return externalEndpoints
 }
 
+func (su *statusUpdater) generateExternalListenerEndpointsFromStatus(status []api_v1.LoadBalancerIngress) []conf_v1alpha1.ExternalEndpoint {
+	var externalEndpoints []conf_v1alpha1.ExternalEndpoint
+	for _, lb := range status {
+		port := su.externalServicePorts
```

Comment:
Here we need to get the port that matches `https` in the LoadBalancer service -- https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/service/loadbalancer.yaml#L14 (assuming we only report IP for Passthrough TransportServers)

Could we do something like we do in `getExternalServicePorts` ? go through the ports of the svc and get the one that matches `https`?   maybe that could be done in a separate function so that the port value is passed to `generateExternalListenerEndpointsFromStatus` as an argument?